src/components/vms/hostvmslist.jsx

```diff
@@ -59,7 +60,7 @@ const VmState = ({ vm, dismissError }) => {
             error={vm.error}
             state={state}
             valueId={`${vmId(vm.name)}-${vm.connectionName}-state`}
-            additionalState={<VmNeedsShutdown vm={vm} />} />
+            additionalState={<><VmNeedsShutdown vm={vm} /><VmUsesSpice vm={vm} /></>} />
```

Comment:
Note: These two can *never* appear at the same time. The "uses spice" warning can only happen on non-running VMs, while "needs shutdown" only on running VMs.